lib/main_common.pm

```diff
@@ -535,7 +535,9 @@ sub load_extra_tests {
         }
         if (check_var('ARCH', 'x86_64')) {
             loadtest "console/docker";
-            loadtest "console/docker_compose";
+            if (check_var('DISTRI', 'opensuse')) {
+                loadtest "console/docker_compose";
+            }
```

Comment:
we should only have these exceptions with a proper reasoning. Compare to the yast2_nis test.

Reply:
I looked at the  [main_common](https://git.io/vdbfY) and also at the [yast2_nis](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/tests/console/yast2_nis.pm) but I am still not sure what do you mean. Could you please be more specific? *e.g. do mean to write a comment in the code?*

AFAIK there's no `docker-compose` package in SLE, so we need to make a distinction between *SLE* and *opensuse*. If there's any preferred way to do this, just let me know.

Reply:
Yes, simply add a comment on top "# no package 'docker-compose' in SLE" and we are good :-)

Reply:
done o>